x-pack/plugins/secops/public/components/timeline/body/renderers/get_row_renderer.test.tsx

```diff
@@ -4,44 +4,33 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
 import { mount } from 'enzyme';
 import { cloneDeep } from 'lodash';
-import { noop } from 'lodash/fp';
 import * as React from 'react';
-import { DragDropContext } from 'react-beautiful-dnd';
-import { Provider as ReduxStoreProvider } from 'react-redux';
-import { ThemeProvider } from 'styled-components';
+import { TestProviders } from 'x-pack/plugins/secops/public/mock/test_providers';
```

Comment:
Another abs path. You can make this relative

Reply:
thanks, that's my editor being "helpful", and me not catching it because i'm looking at the code i'm writing instead of what's being auto-imported